File: Day-6-Lets-Review/main.c

#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int main()
{
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */
    int test_cases;
    char str[10000];
    scanf("%d\n", &test_cases);

    for (int i = 0; i < test_cases; i++)
    {
        // get string input
        scanf("%s", &str);

        for (int i = 0; i < strlen(str); i += 2)
        {
            printf("%c", str[i]);
        }

        printf(" ");

        for (int i = 1; i < strlen(str); i += 2)
        {
            printf("%c", str[i]);
        }
        printf("\n");
    }

    return 0;
}

// gcc Day-6-Lets-Review/main.c -o main ./Day-6-Lets-Review/main
// if ($?) { gcc main.c -o main } ; if ($?) { .\main }

File: Day-6-Lets-Review/main.py
# Enter your code here. Read input from STDIN. Print output to STDOUT 

test_cases = int(input())

for _ in range(test_cases):
    even_str = ""
    odd_str = ""

    str = input("")
    for i in range(len(str)):
        if i % 2 == 0:
            even_str += str[i]
        else:
            odd_str += str[i]

    print(even_str, odd_str)
